### PR TITLE
Make changes to the copy

### LINKED ISSUE #142: Explain groups of teams
It may not be immediately obvious to users what this means:

screen shot 2015-08-25 at 3 48 32 pm

Also, we should be consistent about how we refer to teams and groups. Maybe we should always use the word team, like we did with organization in the organization / classroom terminology.

### LINKED ISSUE #161: Warn students about incoming organization invitation email
When students accept an assignment for the first time they receive an email from GitHub alerting them about their pending organization invitation.

I think we should let the user know that this email will be sent and that it can safely be ignored when we successfully add them to an organization.

### PR DESCRIPTION
**Pull request by tarebyte**
Closes #161, closes #142 

Feedback very much welcome.

/cc @johndbritton

### CODE REVIEW

File: app/controllers/organizations_controller.rb [hunk 1]
```diff
@@ -50,7 +50,7 @@ def update
   def destroy
     if @organization.update_attributes(deleted_at: Time.zone.now)
       DestroyResourceJob.perform_later(@organization)
-      flash_message = "Organization \"#{@organization.title}\" was removed"
+      flash_message = "A job has been queued to remove your organization \"#{@organization.title}\""
```
Comment (johndbritton) [inline app/controllers/organizations_controller.rb]: The user doesn't need to know about jobs or queueing.

> Your organization, @organization, is being removed.

File: app/views/assignment_invitations/successful_invitation.html.erb [hunk 2]
```diff
@@ -14,6 +14,7 @@
 <div class="site-content-body">
   <div class="markdown-body">
     <h2>You are ready to go!</h2>
+    <p>If this is your first assignment, you will recieve an email to join the <%= link_to "@#{decorated_organization.login}", decorated_organization.github_url %>. You can ignore that.</p>
```
Comment (johndbritton) [inline app/views/assignment_invitations/successful_invitation.html.erb]: Can we only show this message in cases where we have added the user to the organization? If not, how about this:

> You may receive an invitation to join `@organization` via email. We have accepted the invitation on your behalf and you can safely ignore the notification email.

File: app/views/assignments/_assignment_form_options.html.erb [hunk 3]
```diff
@@ -7,13 +7,13 @@
   <div class="form-checkbox">
     <label><%= f.radio_button :public_repo, true %> Public</label>
     <%= mega_octicon('repo') %>
-    <span class="note">Anyone can see this repository. You choose who can commit.</span>
+    <span class="note">Anyone can see this repository and it's contents.</span>
```
Comment (johndbritton) [inline app/views/assignments/_assignment_form_options.html.erb]: In this context, we're not talking about a repository. We're talking about all student submissions, maybe this:

> Students will submit their projects using public repositories. All submissions will be visible to the world.

File: app/views/assignments/_assignment_form_options.html.erb [hunk 4]
```diff
@@ -7,13 +7,13 @@
...
   </div>
 
   <div class="form-checkbox">
     <label><%= f.radio_button :public_repo, false %> Private</label>
     <%= mega_octicon('lock') %>
-    <span class="note">You choose who can see and commit to this repository.</span>
+    <span class="note">This is can only be seen by the student, and the Owners of the organization.</span>
```
Comment (johndbritton) [inline app/views/assignments/_assignment_form_options.html.erb]: > Students will submit their projects using private repositories. Submissions will only be visible to the submitter(s) and organization owners.

File: app/views/group_assignment_invitations/show.html.erb [hunk 5]
```diff
@@ -14,24 +14,23 @@
...
-      <dd><%= text_field_tag 'group[title]', nil, placeholder: 'Create a new team', class: 'textfield' %></dd>
-    </dl>
-
     <% if @groups.present? %>
-      <dt><label>OR choose an existing group of teams</label></dt>
+      <dt><label>Join an existing team</label></dt>
       <dl class="form">
         <dd><%= select :group, :id, options_for_select(@groups), include_blank: true %></dd>
       </dl>
     <% end %>
```
Comment (johndbritton) [inline app/views/group_assignment_invitations/show.html.erb]: Maybe we can insert an "OR" here to indicate they have a choice.

File: app/views/group_assignment_invitations/successful_invitation.html.erb [hunk 6]
```diff
@@ -14,6 +14,7 @@
 <div class="site-content-body">
   <div class="markdown-body">
     <h2>You are ready to go!</h2>
+    <p>If this is your first assignment, you will recieve an email to join the <%= link_to "@#{decorated_organization.login}", decorated_organization.github_url %>. You can ignore that.</p>
```
Comment (johndbritton) [inline app/views/group_assignment_invitations/successful_invitation.html.erb]: Same as above.

Can we only show this message in cases where we have added the user to the organization? If not, how about this:

> You may receive an invitation to join `@organization` via email. We have already accepted the invitation on your behalf. No further action is necessary.

File: app/views/group_assignments/_group_assignment_form_options.html.erb [hunk 7]
```diff
@@ -7,13 +7,13 @@
   <div class="form-checkbox">
     <label><%= f.radio_button :public_repo, true %> Public</label>
     <%= mega_octicon('repo') %>
-    <span class="note">Anyone can see this repository. You choose who can commit.</span>
+    <span class="note">Anyone can see this repository and it's contents.</span>
```
Comment (johndbritton) [inline app/views/group_assignments/_group_assignment_form_options.html.erb]: > Submit assignments using public repositories. All submissions will be visible to the world.

File: app/views/assignment_invitations/successful_invitation.html.erb (hunk 2 above)
Comment (tarebyte) [inline app/views/assignment_invitations/successful_invitation.html.erb]: It's not particularly easy to do that, but I think that message is fine.

File: app/views/group_assignments/_group_assignment_form_options.html.erb [hunk 8]
```diff
@@ -7,13 +7,13 @@
...
   </div>
 
   <div class="form-checkbox">
     <label><%= f.radio_button :public_repo, false %> Private</label>
     <%= mega_octicon('lock') %>
-    <span class="note">You choose who can see and commit to this repository.</span>
+    <span class="note">This is can only be seen by the team, and the Owners of the organization.</span>
```
Comment (johndbritton) [inline app/views/group_assignments/_group_assignment_form_options.html.erb]: > Submit assignments using private repositories. Submissions will only be visible to the submitting team members and organization owners.

**Comment by johndbritton** [comment]
On the new assignment page, can you use `participant` instead of `student`?

screen shot 2015-09-02 at 4 07 30 pm

File: app/views/group_assignment_invitations/show.html.erb [hunk 9]
```diff
@@ -13,25 +13,24 @@
...
 
+    <dl class="form">
+      <dt><label><%= 'OR' if @groups.present? %> Create a new team</label></dt>
```
Comment (tarebyte) [inline app/views/group_assignment_invitations/show.html.erb]: @johndbritton I moved the 'Join and existing team' to the top and I added the OR to `Create a new team` only if they have the option of doing one or the other.

Comment (johndbritton) [inline app/views/group_assignment_invitations/show.html.erb]: :+1:

**Pull request merged**